python/ray/actor.py

```diff
@@ -837,6 +842,10 @@ def __del__(self):
                 worker.core_worker.remove_actor_handle_reference(
                     self._ray_actor_id)
 
+
+    def ready(self):
+        return self._actor_method_call("__ray_ready__", num_returns=1)
```

Comment:
would this call be blocked by other tasks in the actor? we do not want it to be queued and blocked in the issue right?

Reply:
@suquark @kfstorm  @ericl Any suggestion for avoiding that blocking?

Otherwise, I'd like to add a separate thread in core worker to execute these specific tasks to avoid queueing in scheduling queue.

Reply:
Could we implement executing `__ray_ready__` in `CoreWorker`? This way this functionality can be reused by Java and C++ APIs, and it also allows non-blocking execution.